src/mui-theme/index.js

```diff
@@ -30,23 +38,54 @@ const LinearProgress = {
   }
 }
 
-const MenuItem = {
+const MuiMenuItem = {
   root: {
-    height: 'auto',
-    '&:hover': {
-      backgroundColor: grayBlueLighter
+    height: 'auto'
+  }
+}
+
+const inputHorizontalSpacing = 10
+
+const MuiInput = {
+  wrapper: {
+    border: {
+      width: 1,
+      style: 'solid',
+      color: palette.text.divider,
+      radius: 5
     },
-    '&:focus': {
-      backgroundColor: grayBlueLighter
-    }
+    padding: [0, inputHorizontalSpacing]
+  },
+  formControl: {
+    marginTop: 5,
```

Comment:
It is only added if an input is used inside of a FormControl component, where you will want to use labels and though will need that margin.